### DISCUSSION TITLE
Toggle Thrust reverse not working?

### DISCUSSION THREAD

**Discussion by nw002** [flybywiresim/aircraft#4298] [Help] [answered]
Since the throttle calibration came into use on the flypad I can't get my joystick throttle to activate the reverse thrust when I have toggled it on the keyboard? It still works with all the other aircraft that have reverse thrust. anyone know how I can get this back as I don't want to spend money on a tca quadrant. Its a real pain when landing on short runways.

**Comment by Paytheon** [comment, marked as answer]
In the Thrustmaster PDF on calibrating the throttle you see they stop the calibration on the 4th step. You need do 5th step calibration too, placing the joysticks in full reverse and pressing the buttons. You'll find this resolves your reverse thruster issues. https://ts.thrustmaster.com/download/accessories/manuals/TCA_Quadrant/TCA_Quadrant-Throttle_Calibration.pdf

**Reply by nw002** [reply]
I am using a conventional joystick (mad Catz) I used to toggle reverse thrust using \ on the keyboard and then the throttle movement would only control reverse thrust until the key was pressed again. Now when using the a32nx experimental version I press the \ key to toggle reverse thrust it has no effect and the joystick remains giving forward thrust. This only happens in the experimental version with calibration in the flypad, the stable and development versions both respond to the \ key press and toggle reverse thrust as do all of the other msfs aircraft that support reverse thrust.

**Reply by aguther** [reply]
Hmm, I catch and use the toggle. In my test it worked.

I can do another check if it's still the case.

**Reply by nw002** [reply]
It seems to be working OK now??? I reset the calibration and then just calibrated the throttle from Idle to togo and now the toggle key switches the throttle to reverse thrust.
